src/main.cc

```diff
@@ -328,6 +329,18 @@ void onExit(void)
     LogEventDeinit();
 }
 
+gboolean terminationSignalHandler(gpointer data) {
```

Comment:
pData as this is a poitner

Reply:
We use { in a new line, we have .clang-format for auto formatting the code, please use it on main.cc before commit or make git do this for you on every commit via a hookor use vscode for this. Created a new issue in relation to this: https://github.com/The-Good-Penguin/tgp-krill-kounter/issues/94

Reply:
Where can I find more information on that clang format? I have installed the clang-format extension in VScode. Do we have some specific format file that I should use?

Reply:
Ah, I see. Thank you for the information.
The formatting along the original file seems a little inconsistent, some statement blocks have the opening curly brace in new line while others don't.